wpimath/src/test/java/edu/wpi/first/math/autodiff/VariableMatrixTest.java

```diff
@@ -0,0 +1,436 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.autodiff;
+
+import static edu.wpi.first.math.MatrixAssertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.ejml.simple.SimpleMatrix;
+import org.junit.jupiter.api.Test;
+
+class VariableMatrixTest {
```

Comment:
Why is this missing the assignment aliasing test? Is `VariableMatrix(VariableBlock)` or `VariableMatrix.set(VariableMatrix)` intended to have the same semantics as the implicitly defined `VariableMatrix& operator=(const VariableMatrix&)`?

Reply:
I ported the test over.